src/python/nimbusml/internal/utils/utils.py

```diff
@@ -300,21 +300,20 @@ def get_clr_path():
     Python 3.x only, as dotnetcore2 is not available for Python 2.x.
```

Comment:
>Python [](start = 57, length = 6)

do we check the python version here? How do we ensure that we fail of Python 2.7?